### DISCUSSION TITLE
DataPLANT Matrix

### DISCUSSION THREAD

**Discussion by Brilator** [nfdi4plants/.github#7] [General]
- [x] Bridge to slack
- [x] unencrypt ARC user support

**Comment by shiltemann** [comment]
Just as an update, the [ARC user support channel](https://matrix.to/#/#arc-user-support:matrix.org) in Matrix has now been bridged to the Düsseldorf/Jülich/Cologne Slack where that community asks their ARC related questions.

This means that any messages sent to either slack or matrix will appear in both places (but you cannot send DMs across platforms)

Unfortunately, this bridging is only possible in a 1:1 fashion, not 1:many, so if other communities already have established support channels and want a similar bridge then we would have to create an additional channel in Matrix. 

And for anybody reading this who would like to join the discussion on Matrix, we currently have 3 channels in the DataPLANT matrix space:
- [DataPLANT general](https://matrix.to/#/#arc-user-support:matrix.org)
- [Data Stewards Exchanges](https://matrix.to/#/#data-steward-exchange:matrix.org)
- [ARC user support](https://matrix.to/#/#dataplant-general:matrix.org)

And everybody is welcome to join!
